x/mongo/driver/operation_legacy.go

```diff
@@ -32,8 +32,15 @@ func (op Operation) getFullCollectionName(coll string) string {
 	return op.Database + "." + coll
 }
 
-func (op Operation) legacyFind(ctx context.Context, dst []byte, srvr Server, conn Connection, desc description.SelectedServer) error {
-	wm, startedInfo, collName, err := op.createLegacyFindWireMessage(dst, desc)
+func (op Operation) legacyFind(
+	ctx context.Context,
+	dst []byte,
+	srvr Server,
+	conn Connection,
+	desc description.SelectedServer,
+	maxTimeMS uint64,
+) error {
```

Comment:
I could be wrong, but it seems like wrapping is more commonplace in our code base:

```go
func (op Operation) legacyFind( ctx context.Context, dst []byte, srvr Server, conn Connection,
	desc description.SelectedServer, maxTimeMS uint64) error {
```